### DISCUSSION TITLE
CMD + K conflict

### DISCUSSION THREAD

**Discussion by tyssen** [trendyminds/craft-palette#31] [Q&A] [answered]
In my usual browser, Firefox, CMD + K does this to my address bar:

![Screenshot 2023-05-18 at 11 03 06 am](https://github.com/trendyminds/craft-palette/assets/370290/ab18a729-e00b-41f8-ac02-39a0dcced33d)

The Craft command palette also opens, but the address bar takes focus so to use the Craft command palette, I then have to click into its search field. 

Interestingly, doing the same thing on Github doesn't launch the address bar search, but instead opens their command palette with the search field focussed. So whatever they're doing is somehow overriding the browser default behaviour.

So my requests would be either:
a) replicate what Github is doing or, 
b) give users the ability to customise the launch keys combination

**Comment by aaronbushnell** [comment, marked as answer]
This is now fixed in 3.1.5, @tyssen. Thanks for reporting!
